backend/users/models.py

```diff
@@ -4,7 +4,11 @@
 User = get_user_model()
 
 
-# Create your models here.
+class LoginGovUser(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    login_id = models.CharField(max_length=100, unique=True)
```

Comment:
[NB] is the ID a 100-char field? I'd imagine this ID from login.gov is a standard size but figure maybe this should be the actual length of whatever it is. (if it's 100, then great!)

Reply:
Yeah, login.gov IDs are UUIDs, so 36 characters long. My tendency in these scenarios, where we're referencing an external field that we don't have control over, is to pad the length a bit in case something about the ID format changes on their end, or if out initial assumptions were somehow incorrect. Thoughts?

Reply:
yeah, I get that, though I also think if you can fit nearly 3 UUIDs in one field, it's probably too big. Also if you specify a size, the DB can allocate that space, which can get inefficient over time

Reply:
Yep that's fair - cut it in half to a max of 50